src/box/lua/console.c

```diff
@@ -326,11 +326,11 @@ lbox_console_readline(struct lua_State *L)
 				rl_on_new_line();
 				rl_replace_line("", 0);
 				lua_pushstring(L, "");
+				lua_pushboolean(L, sigint_called);
 
 				readline_L = NULL;
-				sigint_called = false;
```

Comment:
Deletion of this line now looks redundant, since in this version of the patch we don't lean on the flag value outside of the function anymore.

Reply:
This flag get set to false every time, when we enter the readline function, this line is actually not needed here.

Reply:
I'm a bit afraid about such assumption regarding the global flag. If we'll consume it from some other place, it may show unexpected values.

Reply:
Fixed.

Reply:
Thanks!